clang-tools-extra/clang-doc/Representation.cpp

```diff
@@ -384,5 +384,12 @@ ClangDocContext::ClangDocContext(tooling::ExecutionContext *ECtx,
   }
 }
 
+void ScopeChildren::sort() {
+  std::sort(Namespaces.begin(), Namespaces.end());
```

Comment:
Prefer llvm::sort. https://llvm.org/docs/CodingStandards.html#beware-of-non-deterministic-sorting-order-of-equal-elements